### PR TITLE
Sign up email verification

### PR DESCRIPTION
**Pull request by skystar-p**
이거 머지되면 `secret.py`에 관련 세팅 넣어줘야 함

### CODE REVIEW

File: backend/api/sign_views.py [hunk 1]
```diff
@@ -18,17 +27,58 @@ def sign_up(request):
...
+        code = str(uuid.uuid4())
+        cache.set('verify-account:' + username, code)
+
+        url_code = b64encode(':'.join((username, code)).encode()).decode()
+
+        url = DOMAIN_NAME + 'api/verify_account/{}/'.format(url_code)
+        send_mail(
+            'Web Mighty: Email Verification',
+            '''
+            To activate your account, please click the <a href="{url}">link</a>.
```
Comment (tirr-c) [inline backend/api/sign_views.py]: `<a>`를 사용할 수 없는 환경이 있을 수도 있으니, link라고만 적어놓지 말고 여기에도 url을 그대로 넣자

Comment (skystar-p) [reply backend/api/sign_views.py]: Fixed

**Comment by tirr-c** [comment]
@web-mighty-homu r+

**Comment by web-mighty-homu** [comment]
:pushpin: Commit 560a71a has been approved by `tirr-c`

**Comment by web-mighty-homu** [comment]
:hourglass: Testing commit 560a71a61d746695b18deac2dea3c19f137fee41 with merge fdd4d02992cac1fc9282e62b6914840d552e1a04...

**Comment by web-mighty-homu** [comment]
:sunny: Test successful - [status-travis](https://travis-ci.org/swsnu/swpp17-team2/builds/310749778?utm_source=github_status&utm_medium=notification)
Approved by: tirr-c
Pushing fdd4d02992cac1fc9282e62b6914840d552e1a04 to master...

**Pull request merged**